src/interfaces.ts

```diff
@@ -0,0 +1,139 @@
+export interface KafkaHealthConfig {
+  thresholds?: {
+    consumer?: {
+      errors?: number;
+      lag?: number;
+      stallLag?: number;
+      minMessages?: number;
+    };
+    producer?: {
+      errors?: number;
+      minMessages?: number;
+    };
+  };
+}
+
+export interface CommonKafkaOptions {
```

Comment:
[Optional] Since this can be depending on kafka version and there is a lot of those we might really do some hashmap and point out to kafka properties. There is simply too much and they differ on different versions. There are couple ones that are fairly common.

There is nothing worse than using API for kafka and expecting it to work but then realizing that my server doesn't support it.